rust/ql/test/query-tests/unusedentities/CONSISTENCY/TypeInferenceConsistency.expected

```diff
@@ -0,0 +1,2 @@
+illFormedTypeMention
+| main.rs:403:18:403:24 | FuncPtr |
```

Comment:
This inconsistency is a path to a function pointer type, which doesn't have a type as we don't support function pointers types.